### ISSUE TITLE
Generate a highlighting-only slideshow ?

### ISSUE THREAD

**Issue by dfriquet** [gwen-interpreter/gwen-web#74]
Hello, hello,

Is there any way to have the slideshow generated when only the highlighting screenshots are taken?

Here is my `local.properties`:

```
gwen.web.capture.screenshots=false
gwen.web.capture.screenshots.highlighting=true
```

This way the screenshot generation overhead is limited, capturing only key moments. The screenshots are then available individually as step attachments, but the whole slideshow is not.

The slideshow could be generated if any screenshot is captured, couldn't it?

**Comment by bjuric** [comment]
@dfriquet Done and released in version [2.50.0](https://github.com/gwen-interpreter/gwen-web/releases/tag/v2.50.0)

**Comment by dfriquet** [comment]
Lightning speed, as usual :zap: 

Thanks a lot!
